### ISSUE TITLE
Repo support 'UL'

### ISSUE THREAD

**Issue by caraco1968** [cpanel/elevate#221]
* 2023-03-28 15:30:30 [ERROR] 1 package(s) installed from unsupported YUM repo 'UL_paper_lantern_themes' from /etc/yum.repos.d/UL_paper_lantern_themes.repo
* 2023-03-28 15:30:30 [ERROR] 3 package(s) installed from unsupported YUM repo 'ul' from /etc/yum.repos.d/ul.repo
* 2023-03-28 15:30:30 [ERROR] 3 package(s) installed from unsupported YUM repo 'ul_bluehost' from /etc/yum.repos.d/UL_bluehost.repo
* 2023-03-28 15:30:30 [ERROR] 2 package(s) installed from unsupported YUM repo 'hgdedi' from /etc/yum.repos.d/dedi.repo
* 2023-03-28 15:30:30 [ERROR] 1 package(s) installed from unsupported YUM repo 'ul_auto' from /etc/yum.repos.d/ul_auto.repo
* 2023-03-28 15:30:30 [WARN] *** Elevation Blocker detected: ***
  One or more enabled YUM repo are currently unsupported.

**Comment by NoTick** [comment]
I'm also having issues with 'ul', 'ul_auto', and 'hgdedi' repo's.

[ERROR] 1 package(s) installed from unsupported YUM repo 'ul_auto' from /etc/yum.repos.d/ul_auto.repo
[ERROR] 2 package(s) installed from unsupported YUM repo 'ul' from /etc/yum.repos.d/ul.repo
[ERROR] 7 package(s) installed from unsupported YUM repo 'hgdedi' from /etc/yum.repos.d/dedi.repo

`UL_hostgator_repo.noarch                      1.1-2.el6              @ul
cpanel_addon_WPC.noarch                       1.4-7                  @ul
eig-wpinstall.noarch                          1.0.0-92               @ul_auto
ESO-utils.noarch                              1.2-387.el7            @hgdedi
eig-upcp_control.noarch                       1.0.0-13.el8           @hgdedi
eigid.noarch                                  1.0.0-26.el6           @hgdedi
hg-dedi-firewall.noarch                       1.0.0-19.el7           @hgdedi
hg-dedi-firewall-cpanel-plugin.noarch         1.0.0-19.el7           @hgdedi
postupcp-dedi.x86_64                          1.0.0-71.el7           @hgdedi
sys-snap.noarch                               1.0.3-6                @hgdedi`

**Comment by toddr** [comment]
ul is on the top of our list to review for support in the coming month.

**Comment by mohitgupt** [comment]
Any updates?

**Comment by jack5432109** [comment]
@toddr I'm also having same issues with these repos. Do you have any updates or workaround you could share?
```
[ERROR] 7 package(s) installed from unsupported YUM repo 'hgdedi' from /etc/yum.repos.d/dedi.repo
[ERROR] 1 package(s) installed from unsupported YUM repo 'ul_auto' from /etc/yum.repos.d/ul_auto.repo
[ERROR] 2 package(s) installed from unsupported YUM repo 'ul' from /etc/yum.repos.d/ul.repo
```

These are the installed packages from the repos
```
# yum list installed | grep -E '@hgdedi|@ul_auto|@ul'​
ESO-utils.noarch                              1.2-411.el7            @hgdedi
UL_hostgator_repo.noarch                      1.1-2.el6              @ul
cpanel_addon_WPC.noarch                       1.4-7                  @ul        
eig-upcp_control.noarch                       1.0.0-13.el8           @hgdedi    
eig-wpinstall.noarch                          1.0.0-92               @ul_auto   
eigid.noarch                                  1.0.0-26.el6           @hgdedi    
hg-dedi-firewall.noarch                       1.0.0-19.el7           @hgdedi    
hg-dedi-firewall-cpanel-plugin.noarch         1.0.0-19.el7           @hgdedi    
postupcp-dedi.x86_64                          1.0.0-71.el7           @hgdedi    
sys-snap.noarch                               1.0.3-6                @hgdedi
```

**Comment by Bruno-Oliveira-e-Oliveira** [comment]
I'm blocked in stage four.

**Comment by Bruno-Oliveira-e-Oliveira** [comment]
I remove the packages of 'dedi' and 'ul' with yum and started the elevate process.
I'm now blocked on stage 4 because of 'dedi' package.
Apparently the 'dedi' repository not exist on the version 8 of CentOS in this website.

![Untitled](https://github.com/cpanel/elevate/assets/31875213/52c0e43d-48c5-4edf-9878-9349ebdf158d)

![Untitled2](https://github.com/cpanel/elevate/assets/31875213/bd0532f6-c43e-4559-8203-fb267bafd877)

![Untitled3](https://github.com/cpanel/elevate/assets/31875213/34da8c93-0b9b-4b0b-829f-e6525092dfe6)

**Comment by jack5432109** [comment]
I was able to successfully elevate to AlmaLinux 8 by removing the packages and disabling the unsupported repositories

**Comment by Bruno-Oliveira-e-Oliveira** [comment]
> I was able to successfully elevate to AlmaLinux 8 by removing the packages and disabling the unsupported repositories

It's the same for me.
I removed the package in '/etc/yum.repos.d/dedi.repo' and concluded the elevate process.
At the moment nothing seems wrong.

**Comment by isaacewing** [comment]
> I was able to successfully elevate to AlmaLinux 8 by removing the packages and disabling the unsupported repositories

same here, for anyone else wondering what/how

`yum-config-manager --disable dedi`

if that doesn't work, do this:
`cd /`
`cat /etc/yum.repos.d/dedi.repo` _this is just to check you have the repo and verify the issue_

![dedi-before](https://github.com/user-attachments/assets/a06fdfa0-fd76-43bf-a5e1-f6cf4eb1e4d8)

`vim /etc/yum.repos.d/dedi.repo`
`i` then change enabled to 0

![dedi-after](https://github.com/user-attachments/assets/1dbaa6de-9767-42ea-839f-962b49d78786)

press esc
`:wq`
`/scripts/elevate-cpanel --continue`

![update-complete](https://github.com/user-attachments/assets/aea70d4f-b533-4ddf-9e18-a16b81dc2728)

on my server, it suggests running the following command
`yum -y remove mysql-shell-8.0.38-1.el7.x86_64 mysql-communi
ty-libs-8.0.38-1.el7.x86_64 mysql-community-client-8.0.38-1.el7.x86_64 mysql-community-client-plugins-8.0.38-1.el7.x86_64 cpanel bluehost_theme-3.0-46.el7.noarch alt-openssl-libs-1.0.2k-2.el7.cloudlinux.10.x86_64 tclx-8.4.0-22.el7.x86_64 alt-libssh2-1.8.0-2.e17.x86_64 mysql-community-libs-compat-8.0.38-1.e17.x86_64`

**Comment by toddr** [comment]
elevate-cpanel will continue to block if you have packages installed from disabled repos.

**Comment by toddr** [comment]
We're working with newfold on this soon.

**Comment by toddr** [comment]
This was fixed in #531
